build-system/babel-plugins/babel-plugin-transform-html-template/index.js

```diff
@@ -29,38 +83,20 @@ function optimizeLiteralOutput(templateLiteral) {
   });
 }
 
-module.exports = function(babel) {
-  const {types: t} = babel;
-
+module.exports = function({types: t}) {
   return {
     name: 'transform-html-templates',
     visitor: {
       TaggedTemplateExpression(path) {
-        if (t.isIdentifier(path.node.tag, {name: 'html'})) {
-          const template = optimizeLiteralOutput(path.node.quasi);
-
-          if (template !== null) {
-            const outputArguments = [
-              t.arrayExpression([t.stringLiteral(template)]),
-            ];
-            path.replaceWith(
-                t.callExpression(t.identifier('html'), outputArguments)
-            );
-          }
-        } else if (t.isCallExpression(path.node.tag) &&
-                   t.isIdentifier(path.node.tag.callee, {name: 'htmlFor'})) {
-          const template = optimizeLiteralOutput(path.node.quasi);
+        const {tag} = path.node;
 
-          if (template !== null) {
-            const wrapperMethodArguments = path.node.tag.arguments;
-            const wrapperMethod = t.callExpression(
-                t.identifier('htmlFor'), wrapperMethodArguments
-            );
-            const outputArguments = [
-              t.arrayExpression([t.stringLiteral(template)]),
-            ];
-            path.replaceWith(t.callExpression(wrapperMethod, outputArguments));
-          }
+        if (t.isIdentifier(tag, {name: 'html'})) {
+          replaceExpression(path, t, t.identifier('html'));
+        } else if (t.isCallExpression(tag) &&
```

Comment:
This can be joined with the above:

```suggestion
        if (t.isIdentifier(tag, {name: 'html'}) ||
            (t.isCallExpression(tag) &&
             t.isIdentifier(tag.callee, {name: 'htmlFor'}))) {
          replaceExpression(path, t, tag);
        }
```